### ISSUE TITLE
Bundle Flash into application

### ISSUE THREAD

**Issue by hulbert** [radiant-player/radiant-player-mac#503]
I know there's talk about dropping Flash in #409 but that seems like a long-term goal. If possible, other apps I've used (like the old Rdio app) have bundled Flash but still use Safari/Webkit underneath. This removes the need to do a global install of Flash (i.e. I am trying to avoid installing Flash globally on my system).

**Comment by jacobwgillespie** [comment]
Yeah, that could potentially work...  The number of issues related to using Webkit vs Chrome have been mounting though, so I'd imagine that we will eventually need to move underlying engines (hence https://github.com/radiant-player/radiant-player-electron).  I believe Electron may even bundle Flash itself...

**Comment by MarshallOfSound** [comment]
@jacobwgillespie Just to confirm, electron does NOT bundle flash.  But it comes with built in MP3 supports and spoofs itself as chrome so the HTML5 player works
